src/visualization/types/Graph/view.tsx

```diff
@@ -54,16 +54,26 @@ import {addAnnotationLayer} from 'src/visualization/utils/annotationUtils'
 import {getColorMappingObjects} from 'src/visualization/utils/colorMappingUtils'
 import {isFlagEnabled} from '../../../shared/utils/featureFlag'
 
-interface Props extends VisualizationProps {
+// Selectors
+import {getByID} from 'src/resources/selectors'
+import {updateViewAndVariables} from 'src/views/actions/thunks'
+
+type ReduxProps = ConnectedProps<typeof connector>
+interface OwnProps extends VisualizationProps {
   properties: XYViewProperties
 }
 
+type Props = OwnProps & ReduxProps
+
 const XYPlot: FC<Props> = ({
   properties,
   result,
   timeRange,
   annotations,
   cellID,
+  view,
+  saveViewPropertiesToIDPE,
+  veoOpen,
```

Comment:
when VEO is open, we want to know to decide whether we need to trigger the saving from the graph or not. (time machine will handle it's own saving in the upcoming PR.)